Platform/Views/Shared/Components/UserSettings/Default.cshtml

```diff
@@ -76,7 +76,7 @@
                     <div class="badge-block">
                         <div class="message icons" onclick="OpenNotifications()"></div>
                         <!-- Notification Badge -->
-                        <span id="NotificationCount" class="e-badge e-badge-primary e-badge-notification e-badge-overlap" style="background-color: #1c86c8;">@Notifications.Count</span>
+                        <span id="NotificationCount" class="e-badge e-badge-primary e-badge-notification e-badge-overlap" style="background-color:red;">@Notifications.Count</span>
```

Comment:
This should match the color scheme of the application, red isn't a part of the color palette that we use